### ISSUE TITLE
Allow Rename Files to move additional non-audio files

### ISSUE THREAD

**Issue by marcelpaulo** [quodlibet/quodlibet#2751]
On `Rename Files`, option `Move album art` allows moving a single image file when the audio files are moved to a new directory:
![screenshot_2018-01-28_14-47-39](https://user-images.githubusercontent.com/10074769/35484883-06ee32b6-043e-11e8-955b-c16b3754ecac.png)
For each album, besides file *cover.jpg*, I usually keep other files associated with the album, such as *booklet.pdf*, *booklet.txt* or *lyrics.pdf*, eventually more pdf files with assorted bits of information about the album, and even other jpg files with scanned pages from the album booklet — when they cannot be comfortably put into a pdf file because of wildy varying sizes. It would be fantastic if renaming to a new directory allowed me to move all these non-audio files, too.

quodlibet is particularly elegant in its handling of config options: they don't sprawl, they are concise and make sense. So I don't know what might be the quodlibet way of implementing this, whether simply moving all non-audio files to the new directory, or allowing a regex pattern to select them. I'd risk the 1st option.

Anyway, it would be great to be able to relocate albums to new directories will all their “baggage”.

**Comment by elbeardmorez** [comment]
As per the pull request and subsequent merge #2560 is this not already implemented via the additional (but no doubt undocumented - oops!) config option?
```
[albumart]
search_filenames = front.jpg,back.jpg,cd*.jpg,<artist>.jpg

[rename]
moveart = true
```
note: the option was called 'filenames' originally, but later changed to for clarity

i've just had a look at the code, and would be surprised if it couldn't cope with some abuse (i.e. append `<artist>.pdf` or even just `*.pdf` etc.)

if not i'll take a harder look why

**Comment by marcelpaulo** [comment]
Oh, I only searched in issues, so I didn't see this, thanks for pointing it out, @elbeardmorez ! I'll search more extensively next time I think of raising any issue.

I've just tried:

```
[albumart]
search_filenames = *.pdf,*.jpg,*.txt
```

and renamed an album with these non-audio files: _cover.jpg_, _deleteme.pdf_, _deleteme.txt_. Only **_cover.jpg_** was moved.

**Comment by elbeardmorez** [comment]
Thank you for trying that. Unfortunately I hardcoded:

`IMAGE_EXTENSIONS = ['jpg', 'jpeg', 'png', 'bmp']`  (renamefiles.py:~133)

..in order to limit scope to (..yep) art!

I'll look at pushing another PR, but in the meantime, search for that line and add the extensions there too and it'll work [touches wood].

**Comment by marcelpaulo** [comment]
You're right, it works ! Added `'pdf'` and `'txt'` to `IMAGE_EXTENSIONS`, and now **all** files in `search_filenames` are moved. This is really great ! Thank you so much, @elbeardmorez !

**Comment by bassstorm** [comment]
I think this one also has a sibling in the field of advanced folder handling: #2992.
